src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,8 @@
-const humanCatDogYears = (number) => {};
+const humanCatDogYears = (number) => {
+    const catYears=24+((number-2)*4);
+    const dogYears=24+((number-2)*5);
+    return [number,catYears,dogYears];
+
+};
```

Comment:
This is fine, except there should be an edge condition accounted for of 0 being passed in
```
const humanCatDogYears = (number) => {
    if (number === 0) {
        return [0, 0, 0];
    }
    const catYears = 24 + (number - 2) * 4;
    const dogYears = 24 + (number - 2) * 5;
    return [number, catYears, dogYears];
};
```